### ISSUE TITLE
Model checkpoints

### ISSUE THREAD

**Issue by michaeltian108** [microsoft/XPretrain#31]
Excellent work!
I'm trying to deploy some attacks on your models, but I cannot fine-tune your pretrained ones on my local server due to a VRAM shortage. Could you please provide the model checkpoint for better accessibility of model applications? Thank you.
